eng/targets/Helix.Common.props

```diff
@@ -43,7 +43,7 @@
   <ItemGroup Condition="'$(TargetArchitecture)' == 'arm64' AND '$(IsHelixDaily)' == 'true' AND '$(IsWindowsOnlyTest)' != 'true'">
     <HelixAvailableTargetQueue Include="(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036" Platform="Linux" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetArchitecture)' == 'arm64' AND '$(IsHelixDaily)' == 'true'">
+  <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'arm64' AND '$(IsHelixDaily)' == 'true'">
```

Comment:
Is there an issue tracking this part? The initial error looked like an easy fix in TestMatrix.

Reply:
I don't believe so, @halter73 could you please confirm? If not, I'll create a new one.

Reply:
Thanks @TanayParikh, but I got it. https://github.com/dotnet/aspnetcore/issues/26872